inc/class-walkers.php

```diff
@@ -36,7 +36,8 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 		}
 	}
 	public function end_el( &$output, $item, $depth = 0, $args = array(), $id = 0 ) {
-		if ( in_array( 'has_children', $item->classes ) ) {
+		$classes = empty( $item->classes ) ? array() : (array) $item->classes;
```

Comment:
Let's add a comment here describing what the code is trying to achieve. Likewise, it might be useful to add a docstring comment to the function.